tango/integrations/beaker/executor.py

```diff
@@ -460,7 +460,8 @@ def _execute_sub_graph_for_step(
                         # We don't need to handle logs from Tqdm since that would result in
                         # duplicate messages (those Tqdm lines get printed directly to the output as well).
                         logging.getLogger(log_record.name).handle(log_record)
-                except JSONDecodeError:
+                except:  # noqa: E722
```

Comment:
I think at the very least we should catch `Exception`, so that `KeyboardInterrupt`, `OutOfMemory`, and stuff like that doesn't get swallowed up.

Reply:
Good pointd: 4089ac